peachjam/templates/peachjam/judgment_detail.html

```diff
@@ -47,4 +47,17 @@
       </dd>
     {% endif %}
   {% endwith %}
+  {% with document.attorneys.all as attorneys %}
+    {% if document.attorneys %}
+      <dt>
+        {% trans 'Attorneys' %}
+      </dt>
+      <dd class="text-muted">
+        {% for attorney in attorneys %}
+          {{ attorney.name }}
```

Comment:
The next line must be on the same as this line otherwise there will be a space between the name and the comma.